requirements/dev.txt

```diff
@@ -56,7 +56,7 @@ cached-property==1.5.2
     #   -c requirements/ci.txt
```

Comment:
The requirement files should not be changed unless the `*.in` files are changed. The `.in`-files control the libraries that are required.